Adding football api fetcher

verigos opened the pull request.

bamertl commented on bet-it/bank/pom.xml:
```diff
@@ -71,8 +71,11 @@
         </dependency>
-
-
+        <dependency>
```
delete this

bamertl commented on bet-it/bet-platform/pom.xml:
```diff
@@ -54,8 +54,16 @@
-
-
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```
delete those

bamertl commented on bet-it/game-master/pom.xml:
```diff
@@ -57,7 +57,16 @@
             <artifactId>lombok</artifactId>
         </dependency>
-
+        <dependency>
```
delete those

bamertl commented on bet-it/game-master/src/main/java/ch/unisg/kafka/spring/controllers/KafkaController.java:
```diff
@@ -0,0 +1,144 @@
+    private Environment environment;
+
+//    private final ProducerService<Game> producerService;
+
```
what is this used for?

bamertl commented on bet-it/shared-domain/pom.xml:
```diff
@@ -0,0 +1,44 @@
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```
what is this module for, it's not used anywhere as far as i can see

bamertl commented on bet-it/shared/src/main/java/ch/unisg/ics/edpo/shared/kafka/KafkaMapProducerFactory.java:
```diff
@@ -1,41 +1,41 @@
-package ch.unisg.ics.edpo.shared.kafka;
```
Undo this

bamertl commented on bet-it/zeebe-addon/pom.xml:
```diff
@@ -98,7 +98,16 @@
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```
undo this

bamertl commented on bet-it/api2kafka/.mvn/wrapper/maven-wrapper.jar:
Adding this at the beginning would have been a smart move, now however this is fairly risky to the other packages.
I don't think we should include a mvn wrapper one hour before the deadline.

verigos committed:
adding api2kafka application

the app fetches on a schedule data for the live-football api
and publishes them to kafka
